docs/content/preview/manage/_index.md

```diff
@@ -11,7 +11,6 @@ menu:
   preview:
     identifier: manage
     parent: launch-and-manage
-    weight: 900
```

Comment:
Just making sure this is deliberate. Removing this because the order is manually specified in menus.toml, right?

Reply:
Deploy doesn't have a weight, and removing this fixes the order. They aren't in menus.toml. Alphabetic perhaps.